xml/System.Collections.Concurrent/ConcurrentDictionary`2.xml

```diff
@@ -586,14 +586,22 @@
       </Parameters>
       <Docs>
         <param name="key">The key of the element to add.</param>
-        <param name="valueFactory">The function used to generate a value for the key</param>
-        <summary>Adds a key/value pair to the <see cref="T:System.Collections.Concurrent.ConcurrentDictionary`2" /> by using the specified function, if the key does not already exist.</summary>
-        <returns>The value for the key. This will be either the existing value for the key if the key is already in the dictionary, or the new value for the key as returned by valueFactory if the key was not in the dictionary.</returns>
+        <param name="valueFactory">The function used to generate a value for the key.</param>
+        <summary>Adds a key/value pair to the <see cref="T:System.Collections.Concurrent.ConcurrentDictionary`2" /> by using the specified function if the key does not already exist, or returns the existing value if the key exists.</summary>
+        <returns>The value for the key. For more information, see the Remarks section.</returns>
```

Comment:
We don't usually need to say this in the return section. Just add the  extra info in the remarks. (and it also doesn't help if you're reading this in your IntelliSense so think about what can be helpful to the customer if all they have is IntelliSense - summary, parameters, return value)